src/dbus_api/macros.rs

```diff
@@ -42,3 +42,9 @@ macro_rules! get_mut_pool {
         }
     };
 }
+
```

Comment:
I think it'll help if there's a comment here, something like:

```// Format a UUID for transport on the D-Bus.```

Just two slashes, we found out eventually that doc comments on macros don't make sense.

Reply:
Rebased and resolved.